Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ListViewExtensions/ListViewExtensionsPage.xaml

```diff
@@ -22,6 +22,7 @@
         <ListView 
             x:Name="SampleListView"
             Margin="12"
+            IsItemClickEnabled="True"
             ItemTemplate="{StaticResource NormalTemplate}"
             extensions:ListViewExtensions.AlternateColor="#33AAAAAA"
```

Comment:
Don't forget to add the extension here too, that's the main one to make sure it doesn't get optimized out by the .NET Native compiler.

Reply:
Sure, will get it done :)